runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -203,7 +205,11 @@ public void onAnimationChanged(AnimationChanged event)
 			/* Magic */
 			case MAGIC_CHARGING_ORBS:
 			case MAGIC_LUNAR_STRING_JEWELRY:
-			case MAGIC_LUNAR_BAKE_PIE:
+			case MAGIC_LUNAR_SHARED:
```

Comment:
Move this case before IDLE, and duplicate the resetTimers(); and lastAnimation = animation; code there